packages/financial-templates-lib/src/price-feed/UniswapPriceFeed.js

```diff
@@ -26,14 +39,26 @@ class UniswapPriceFeed extends PriceFeedInterface {
     this.getTime = getTime;
     this.historicalLookback = historicalLookback;
     this.invertPrice = invertPrice;
+    // The % of the lookback window (historicalLookback + twapLength) that we want to query for Uniswap
+    // Sync events. For example, 1.1 = 110% meaning that we'll look back 110% * (historicalLookback + twapLength)
+    // seconds, in blocks, for Sync events.
+    this.bufferBlockPercent = 1.1;
+
+    // TODO: Should/Can we read in `poolDecimals` from the this.uniswap?
+    this.poolPrecision = poolDecimals;
+    this.decimals = priceFeedDecimals;
```

Comment:
nit: can you store this variable as `this.priceFeedDecimals` for consistency to be explicit what kind of decimals you are referring to?

Reply:
Done and did the same for `BalancerPriceFeed`